add-on/src/lib/constants.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 /* eslint-env browser, webextensions */
 
-exports.welcomePage = '/dist/landing-pages/welcome/index.html'
-exports.optionsPage = '/dist/options/options.html'
-exports.tickMs = 250 // no CPU spike, but still responsive enough
+export const welcomePage = '/dist/landing-pages/welcome/index.html'
+export const optionsPage = '/dist/options/options.html'
+export const tickMs = 250 // no CPU spike, but still responsive enough
```

Comment:
I would love a JSdoc comment on this to explain its intention. I can guess, but it's better if we don't force others to.